### PR TITLE
bpo-12217 Added linking of pdb, faulthandler and traceback documentation

### PR DESCRIPTION
**Pull request by jdkandersson**
https://bugs.python.org/issue12217

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Review by zooba — APPROVED** [review]
Once the CLA is approved (it's already been submitted), this is ready to merge.

File: Doc/library/faulthandler.rst [hunk 1]
```diff
@@ -170,3 +173,8 @@ handler:
       File "<stdin>", line 1 in <module>
     Segmentation fault
 
+
+.. seealso::
```
Comment (vstinner) [inline Doc/library/faulthandler.rst:177]: Would it be possible to put it at the top, rather than at the end? Same remark for pdb.rst.

**Comment by matrixise** [comment]
@jdkandersson We can't use your PR because the bot does not want to accept your PR. A problem with the user of your commits, this user has not signed the CLA (david-andersson) but the CLA is signed for @jdkandersson. Please could you close this PR and recreate a new PR where you use the right github account (jdkandersson) and not (david-andersson) for your commits and PR.

Thank you

**Comment by csabella** [comment]
@jdkandersson, please address the prior comments regarding the CLA.  Thank you!

**Pull request closed without merging**